tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -390,14 +395,11 @@ async def federated_aggregate(self, arg):
   async def federated_sum(self, arg):
     py_typecheck.check_type(arg.type_signature, computation_types.FederatedType)
     zero, plus = tuple(await asyncio.gather(*[
-        executor_utils.embed_tf_scalar_constant(
-            self.federating_executor,
-            arg.type_signature.member,
-            0),
+        executor_utils.embed_tf_scalar_constant(self.federating_executor,
+                                                arg.type_signature.member, 0),
```

Comment:
Looks like the this PR is matching the linting in [`tensorflow/federated`](https://github.com/tensorflow/federated/blob/688938cca43112ae9820f2faf12336406a3f6d27/tensorflow_federated/python/core/impl/executors/federating_executor.py#L724) repo. Not sure what happened on master in `tfe/federated`.

Reply:
apparently yapf is not deterministic, so it could simply have arrived at different strategies (which was one of the reasons we went with black) -- don't know if that's what happened here